src/hooks/graphqlState/useEntryState.tsx

```diff
@@ -39,11 +39,12 @@ export const useEntryState = (
   projectId: number,
   entryId?: number | string,
   options?: QueryHookOptions<TEntryData, TEntryVariables>,
+  entryTemplate: Entry = {} as Entry,
 ) => {
   const { toast } = useNotification()
 
-  const [entry, setEntry] = useState<Entry>({} as Entry)
-  const [baseEntry, setBaseEntry] = useState<Entry>({} as Entry)
+  const [entry, setEntry] = useState<Entry>(entryTemplate as Entry)
```

Comment:
Aren't you double casting here, if the entryTemplate is already casted to Entry above?

Reply:
Hmm this is not supposed to be here